Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.Model.sb.in

```diff
@@ -25,3 +25,27 @@
 
 (allow mach-lookup
     (global-name "com.apple.CARenderServer"))
+
+(allow iokit-open-user-client
+    (iokit-user-client-class "AppleParavirtDeviceUserClient"))
+(allow iokit-open-user-client
+    (iokit-user-client-class "AGXDeviceUserClient"))
+
+(allow mach-lookup
+    (global-name "com.apple.audio.AudioSession"))
```

Comment:
Is access to this audio service required?

Reply:
Not for our current scope. I have removed this!